olive/model/handler/pytorch.py

```diff
@@ -65,6 +65,7 @@ def __init__(
         adapter_path: OLIVE_RESOURCE_ANNOTATIONS = None,
         model_attributes: Optional[Dict[str, Any]] = None,
         mlflow_transformer_model_cache_dir: Optional[str] = None,
+        generative: bool = False,
```

Comment:
this also needs to be added to `json_config_keys` so that it is added to the json serialization https://github.com/microsoft/Olive/blob/13a0faff67e9f3a1054aa822d815989dfceda3d7/olive/model/handler/pytorch.py#L48

Reply:
Done!